### PR TITLE
Consolidate link text documentation

### PR DESCRIPTION
**Pull request by jdbaldry**
There's docs all over the place for links:

- https://grafana.com/docs/writers-toolkit/write/links/
- https://grafana.com/docs/writers-toolkit/write/useful-link-text/
- https://grafana.com/docs/writers-toolkit/write/style-guide/style-conventions/#links-and-references
- https://grafana.com/docs/writers-toolkit/write/markdown-guide/#links

I'm open to ideas for how to better consolidate this information.

### CODE REVIEW

**Review by clayton-cornell — APPROVED** [review]
Looks good.

File: docs/sources/write/style-guide/useful-link-text/index.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+- Don't use words such as _Click here_, _More_, and _Read more_ as standalone link text, because they can be confusing when a screen reader reads them out of context.
+
+- When linking to a GitHub repository, include `repository` in the sentence text or link text.
+
+  For example:
+
+  - To learn more, go to the `[faro-web-sdk]()` repository.
+  - To learn more about Grafana Faro, go to its `[repository]()`.
+
+- When linking from outside of a documentation page, include `documentation` in the text.
```
Comment (clayton-cornell) [inline docs/sources/write/style-guide/useful-link-text/index.md:31]: Do we ever link from outside a doc page? On blogs maybe? Or?

Comment (jdbaldry) [reply docs/sources/write/style-guide/useful-link-text/index.md:31]: Yeah, I believe that's the intention of this sentence. If you are writing a blog or product page link to the documentation.

**Comment by JStickler** [comment]
Wouldn't it make more sense to nest "Writing useful link text" directly under the "Links" topic rather than moving it under the Style Guide?  I know it's essentially style information, but putting it under the "Links" topic would make it obvious that we provide additional guidance on how to write links.

**Comment by jdbaldry** [comment]
> Wouldn't it make more sense to nest "Writing useful link text" directly under the "Links" topic rather than moving it under the Style Guide? I know it's essentially style information, but putting it under the "Links" topic would make it obvious that we provide additional guidance on how to write links.

Yeah that makes a lot of sense to me.
It means one less page about links :)

**Comment by JStickler** [comment]
>It means one less page about links 

I wasn't even thinking about combining the pages. But you're right, put it all on one page!

**Comment by jdbaldry** [comment]
> > It means one less page about links
> 
> I wasn't even thinking about combining the pages. But you're right, put it all on one page!

Oh haha, I misunderstood :) Both make sense. I read what I wanted to hear I guess!

**Comment by jdbaldry** [comment]
Looking again at the length of the page, I think it makes sense to be a separate page under the links topic.

**Comment by clayton-cornell** [comment]
@jdbaldry Is this one ready to merge?

**Comment by jdbaldry** [comment]
> @jdbaldry Is this one ready to merge?

Yeah, let's get this in.

**Pull request merged**